### ISSUE TITLE
npm module is using var React = require('react-native')

### ISSUE THREAD

**Issue by andyfen** [dancormier/react-native-swipeout#125]
it looks ok in the github repo, but i just downloaded from npm and the module is requiring React from the native package rather than react

var React = require('react-native')
in the index.js file

**Comment by lewisbarber** [comment]
+1

**Comment by edo1493** [comment]
This component supports 0.24+, that's probably the reason why you are getting that. What version of RN are you on?

**Comment by andyfenelon** [comment]
"react-native": "0.30.0", 
it's because the npm module hasn't been updated

**Comment by edo1493** [comment]
Yep

**Comment by edo1493** [comment]
In case you need it, go into the index.js and add: 

```
var React = require('react')
var ReactNative = require('react-native');
var {PanResponder, TouchableHighlight, StyleSheet, Text, View} = ReactNative;
```

**Comment by YuriyUlantsev** [comment]
npm i --save https://github.com/dancormier/react-native-swipeout.git

**Comment by jseminck** [comment]
Any update on this? Is it possible to add a new contributor who can publish this to NPM?

**Comment by digitalmaster** [comment]
Any movement here?

**Comment by silentcloud** [comment]
I have already published version `2.0.13`
